_api-reference/ingest-apis/processors-reference/append.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: default
+title: Append
+parent: Ingest processors 
+grand_parent: Ingest APIs
+nav_order: 10
+---
+
+# Append
+
+The `append` processor is used to add values to a field:
+- If the field is an array, the `append` processor appends the specified values to that array.
+- If the field is a scalar field, the `append` processor converts it to an array and appends the specified values to that array.
+- If the field does not exist, the `append` processor creates an array with the specified values.
+
+The syntax for the `append` processor is: 
+
+```json
+{
+    "append": {
+        "field": "field_name",
+        "value": ["value1", "value2", "{{value3}}"]
+    }
+}
+```
+
+## Parameters
+
+The following table lists the required and optional parameters for the `append` processor.
+
+**Parameter** | **Required** | **Description** |
+|-----------|-----------|-----------|
+`field`  | Required  | Name of the field where the data should be appended. Supports template snippets.|
+`value`  | Required  | Value to be appended. This can be a static value, a dynamic value derived from existing fields, or a value obtained from external lookups. Supports template snippets. |
```

Comment:
Do you know how to obtain value from external lookup? Link to an example would be nice.